### PR TITLE
AboutUs and Readme.md: Acknowledge GSoC2018 #8805

### LINKED ISSUE #8805: AboutUs and Readme.md: Acknowledge GSoC2018
There are two issue to solve 
1) under the banner of Google summer of codes "and" is repeated twice 
2) Refering GSoc 2018 

This could be decent first timer issue

### PR DESCRIPTION
**Pull request by yatri1609**
Fixes #8805 

**PR Checklist**

Ensure that you have:
- [x] Read and understood our PR guideline: https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr
  - [x] Added the issue number to the "Fixes" keyword above
  - [x] Titled the PR as specified in the abovementioned document
- [x] Made your changes on a branch other than `master` and `release`
- [x] Gone through all the changes in this PR and ensured that:
  - [x] They addressed one (and only one) issue
  - [x] No unintended changes were made
- [x] Run and passed static analysis: `./gradlew lint` and `npm run lint`
- [x] Added/updated tests, if changes in functionality were involved
- [x] Added/updated documentation to public APIs (classes, methods, variables), if applicable

**Outline of Solution**

### CODE REVIEW

**Comment by yatri1609** [comment]
Ready for review.

**Comment by damithc** [comment]
@yatri1609 w.r.t. PR title, instead of doing trial and error, a better approach is to look at existing PRs to see if you are following the correct format :-)

**Comment by tshradheya** [comment]
@yatri1609 How about updating `Readme.md` to mention about GSoC 2018 as well?

**Comment by yatri1609** [comment]
@tshradheya  Ya sure I would do that as well.

**Comment by yatri1609** [comment]
Ready for review.

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ TEAMMATES team wishes to thank the following invaluable contributions:
 * [**School of Computing, National University of Singapore (NUS)**](http://www.comp.nus.edu.sg), for providing us with the infrastructure support to run the project.
 * [**Centre for Development of Teaching and Learning (CDTL)**](http://www.cdtl.nus.edu.sg/) of NUS, for supporting us with several *Teaching Enhancement Grants* over the years.
 * **Learning Innovation Fund-Technology (LIF-T)** initiative of NUS, for funding us for the 2015-2018 period.
-* **Google Summer of Code** Program, for including TEAMMATES as a mentor organization in *GSoC2014*, *GSoC2015*, *GSoC2016*, and *GSoC2017* editions.
+* **Google Summer of Code** Program, for including TEAMMATES as a mentor organization in *GSoC2014*, *GSoC2015*, *GSoC2016*, *GSoC2017*, and *GSoC2018* editions.
```
Comment (tshradheya) [inline README.md]: Lets remove the unnecessary `,` after `GSoC2017`

**Comment by yatri1609** [comment]
Ready for review.

File: README.md (hunk 1 above)
Comment (whipermr5) [reply README.md]: Actually the [Oxford comma](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/) can be [useful](https://www.theguardian.com/books/2017/mar/16/oxford-comma-helps-drivers-win-dispute-about-overtime-pay), though I guess in this case it doesn't really make a difference.

**Pull request merged**